src/qutip_cupy/__init__.py

```diff
@@ -36,7 +36,19 @@
         (data.Dense, CuPyDense, cd.dense_from_cupydense),
     ]
 )
-data.to.register_aliases(["cupyd"], CuPyDense)
+data.to.register_aliases(["cupyd", "CuPyDense"], CuPyDense)
+
+
+def is_cupydense(data):
+    return isinstance(data, CuPyDense)
+
+
+def is_cupy(data):
+    return isinstance(data, cp.ndarray)  # noqa: F821
+
+
+data.create.add_creators([(is_cupydense, CuPyDense, 80), (is_cupy, CuPyDense, 81)])
```

Comment:
I thought we decided not to have the `is_cupydense` creator? Or is it useful for something? There is also no need to have a priority of `81` rather than `80` -- only one of `is_cupydense` and `is_cupy` can return True at a time.

Reply:
Ah, I see I misunderstood your last comment. Dropping it then

Reply:
Thanks!